### PR TITLE
23 add legend and guidelines for using the app

### PR DESCRIPTION
**Pull request by yunjin08**
lesgo bitch

### CODE REVIEW

File: components/LegendPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        'borderClass': 'border-custom-red',
+        'bgClass': 'bg-custom-red'
+    },
+    {
+        'title': 'Start State',
+        'description': 'F',
+        'borderClass': 'border-custom-green',
+        'bgClass': 'bg-custom-green'
+    }
+];
```
Comment (maxellmilay) [inline components/LegendPanel.tsx:34]: @yunjin08 I suggest we put this in a separate file, inside a folder named constants

File: components/LegendPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+const guideLine = [
+    'Concatenation: Represented by . (e.g., a.b means ab)',
+    'Union (Or): Represented by | (e.g., a|b means either a or b)',
+    'Kleene Star: Represented by * (e.g., a* means zero or more occurrences of a)',
+    'When checking a string, the DFA visualizer only accepts strings composed of a and b, \
+    ignoring any special characters. For instance, while a.b is valid for DFA generation, the string should be entered as ab for checking.'
+]
```
Comment (maxellmilay) [inline components/LegendPanel.tsx:42]: @yunjin08 also put this in a constants file

**Review by maxellmilay — CHANGES_REQUESTED** [review]
Please address the comments, thank you!

File: components/LegendPanel.tsx (hunk 1 above)
Comment (yunjin08) [reply components/LegendPanel.tsx]: @maxellmilay done aydol

**Review by maxellmilay — APPROVED** [review]
@yunjin08 LGTM :fire:

**Pull request merged**